src/Core/Services/Implementations/UserService.cs

```diff
@@ -973,6 +973,9 @@ await _referenceEventService.RaiseEventAsync(
             await paymentService.CancelAndRecoverChargesAsync(user);
             throw;
         }
+
```

Comment:
You can run `git checkout main src/Core/Services/Implementations/UserService.cs` so this file won't show up in your PR, as the changes are only whitespace.